release/tools/msetdg.py

```diff
@@ -7,12 +7,12 @@
 
 try:
 	fileName = "msetdg.bin";
-	tid = [0x00020000, 0x00021000, 0x00022000] #JPN, USA, EUR
-	tver = [0x1A, 0x1F, 0x19]
+	tid = [0x00020000, 0x00021000, 0x00022000, 0x00026000, 0x00027000, 0x00028000] #JPN, USA, EUR, CHN, KOR, TWN
+	tver = [0x1A, 0x1F, 0x19, 0x06, 0x04, 0x05]
```

Comment:
Last time I checked, only CHN consoles use different ROP offsets for the MSET exploit. KOR and TWN should be fine, though, but it never hurts to check. We still need to work on the 9.x ROP chain, btw.